packages/suite/src/support/messages.ts

```diff
@@ -5426,6 +5426,11 @@ export default defineMessages({
         defaultMessage: 'Not enough {symbol} to cover transaction fee',
         id: 'AMOUNT_NOT_ENOUGH_CURRENCY_FEE',
     },
+
+    AMOUNT_NOT_ENOUGH_CURRENCY_FEE_WITH_ETH_AMOUNT: {
+        defaultMessage: 'Not enough {symbol} to cover transaction fee {feeAmount}',
```

Comment:
How about
`Not enough {symbol} to cover transaction fee ({feeAmount} {symbol)',`

Reply:
If it is ok, I made a little refactoring. Commit: https://github.com/trezor/trezor-suite/pull/15993/commits/0d79d6dff1cd5b390161fc528e10e65153248453 More information is in the related PR.